crud/common/utils.lua

```diff
@@ -33,10 +37,36 @@ end
 
 function utils.get_space(space_name, replicasets)
     local replicaset = select(2, next(replicasets))
-    local space = replicaset.master.conn.space[space_name]
+
+    local master = replicaset.master
+    if not master.conn:ping({timeout = ping_timeout}) then
```

Comment:
So, we'll do an extra PING call on each crud operation?
Although it is a working solution, we need to measure the performance impact. What about pinging once, saving the schema internally and update it when the schema version changes, as it is done inside the connection?
Also, it is possible to fetch schema from replicas too. Probably it may be faster and save time for the master a bit.